### ISSUE TITLE
Tool-bar on the left position messes up Tree View right border position on rezise

### ISSUE THREAD

**Issue by murilomentor** [suda/tool-bar#73]
With the tool-bar positioned on the left, when I try to re-size the Tree View pane, the right border of the Tree View, which is being dragged, moves from under the mouse pointer to the right, possibly by the same amount of the tool-bar size.

I'm using: 
Atom 0.210.0
tool-bar 0.1.7
tool-bar-main 0.0.8

![Tree View resizing issue](https://dl.dropboxusercontent.com/u/6244657/Atom_tool-bar_issue.gif)

**Comment by jerone** [comment]
Noticed this too before. Actually this is a Atom issue, but haven't had the time to report a proper bug report.
```
var x = document.createElement('div');
x.appendChild(document.createTextNode('test'));
atom.workspace.addLeftPanel({item:x,priority:10});
```

**Comment by jerone** [comment]
Upstream: https://github.com/atom/tree-view/issues/533

**Comment by jerone** [comment]
Fixed by atom/tree-view#534
